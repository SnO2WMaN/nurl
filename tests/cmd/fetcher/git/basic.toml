args = [
    "https://github.com/nix-community/nurl",
    "v0.3.0",
    "--fetcher",
    "fetchgit",
]
