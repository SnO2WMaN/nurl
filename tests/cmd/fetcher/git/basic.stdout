fetchgit {
  url = "https://github.com/nix-community/nurl";
  rev = "v0.3.0";
  hash = "sha256-jZ+cCp1THDhfHH5yMmRPjGuthOqsgcF/3OjZ61FMdA4=";
}